server/src/main/resources/application.yaml

```diff
@@ -13,12 +13,9 @@ elasticsearch:
   serverUrl: http://localhost:9200
   apiKey: <sample-api-key>
   search_as_you_type:
-    record_suggest:
-      path: record_suggest
-      fields: abstract_phrases  # if you have multiple search_as_you_type fields, separate them with commas e.g title, description
-    parameter_vocab:
-      path: parameter_vocab
-      field: label
+    search_suggestions:
```

Comment:
Need change to https://github.com/aodn/appdeploy/blob/main/tg/edge/ogcapi-java/ecs/variables.yaml and other env?

Reply:
I'm not sure if it will be straight forward to rename a remote branch than already has a reviewed PR, I might just rename the PR name this time.